tests/unit/HeadlessLayerImplicitAnimationTests.swift

```diff
@@ -83,67 +132,125 @@ class HeadlessLayerImplicitAnimationTests: XCTestCase {
   }
 
   func testCATransactionTimingTakesPrecedenceOverUIViewTimingInside() {
```

Comment:
This looks like the same as `testImplicitlyAnimatesInUIViewAnimateBlockWithCATransactionDuration`. Am I missing something?

Reply:
Removed.